charts/authz/templates/Service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Values.name }}-svc
+  namespace: {{ .Release.Namespace }}
+spec:
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: 80
+  selector:
+    app: {{ .Values.labels.app }}
```

Comment:
we set those labels in a lot of different places. Can we somehow DRY the code and make 1 var with those labels and use them everywhere